content/webapp/components/Image/Image.tsx

```diff
@@ -0,0 +1,25 @@
+import { FilledImageFieldImage } from '@prismicio/types';
+import { classNames } from '@weco/common/utils/classnames';
+import { getImageUrlAtSize } from '../../services/prismic/images';
+
+type Props = {
+  image: FilledImageFieldImage;
+};
+
+const Image = ({ image }: Props) => {
```

Comment:
Is it worth putting a comment about the intention with this new image class?

Reply:
Nice idea, I have also renamed it to Prismic image to avoid early abstractions.